code/modules/paperwork/paper_premade.dm

```diff
@@ -104,3 +104,8 @@
 /obj/item/paper/fluff/stations/lavaland/orm_notice
 	name = "URGENT!"
 	info = "A hastily written note has been scribbled here... <br><br> Please use the ore redemption machine in the cargo office for smelting. PLEASE! <br><br>--The Research Staff"
+
+//////////// GlowStation
+/obj/item/paper/fluff/stations/glowstation/vault
+	name = ""
+	info = "<h1>Notice</h1> <p>From the department of nuclear accounting™.</p> <p>To all who it may apply.<br>The station’s nuclear self destruct device.<br>And any local <em>secret</em> documentation.</p> <p>May be found at the station’s nuclear site.<br>Located north of engineering on deck 1.<br>Accessed via the command staircase on the northen side of the command hallway.</p> <p>signed: <strong><em>Kevin Firewind</em></strong><br>From nuclear accounting™.</p> <p>Glory to Nanotrasen™</p>"
```

Comment:
This has no name?
Also, please use multiline strings, you can end a line in `\` and it will allow you to split the string onto the following line.
Also your apostrophes here are `’` when they should be `'`